XDR - host_ip at incident alerts (#10173)
demisto/content: Packs/CortexXDR/ReleaseNotes/2_4_13.md

#### Integrations
##### Palo Alto Networks Cortex XDR - Investigation and Response

> - Added support for the **xdr-get-incident-extra-data** command which returns a list of host_ip's, at Incident Alerts. 

Reviewer: ```suggestion
- Added support for host_ip's list in the data returned from **xdr-get-incident-extra-data** command.
```
